app/src/main/scala/org/alephium/explorer/persistence/queries/TokenQueries.scala

```diff
@@ -178,4 +195,166 @@ object TokenQueries extends StrictLogging {
     """
       .paginate(pagination)
       .asAS[(TokenId, Option[U256], Option[U256])]
+
+  def insertFungibleTokenMetadata(
+      token: TokenId,
+      metadata: FungibleTokenMetadata
+  ): DBActionW[Int] = {
+    sqlu"""
+      INSERT INTO fungible_token_metadata (
+        "token",
+        "name",
+        "symbol",
+        "decimals"
+        )
+      VALUES (${token},${metadata.name},${metadata.symbol},${metadata.decimals})
+      ON CONFLICT
+      ON CONSTRAINT fungible_token_metadata_pkey
+      DO NOTHING
```

Comment:
Should we constraint on the column name we create `token` instead of the database generated name `fungible_token_metadata_pkey`?

```sql
VALUES ...
      ON CONFLICT(token)
      DO NOTHING
```

Or

```sql
ON CONFLICT
DO NOTHING
```

Reply:
oh I admit I used what I found on psql documentation. If your second proposition is equivalent I'm totally in!

Reply:
I did try and added a test in [b9b49c3](https://github.com/alephium/explorer-backend/pull/486/commits/b9b49c360dd2306d535aec6f3d50f18fc712748b) let me know what you think